src/Sylius/Bundle/ApiBundle/Resources/config/api_platform/resources/TaxonImage.xml

```diff
@@ -227,4 +171,89 @@
         </operations>
     </resource>
 
+    <resource class="Sylius\Component\Core\Model\Taxon">
```

Comment:
Hmm, I think this configuration should be in the `Taxon.xml` file

Reply:
I'm not sure if this is the best place, but I would leave it here, as it's an endpoint that adds new images and the use of `Taxon` resource is just our workaround 🤔